tests/test_detection_pytest.py

```diff
@@ -0,0 +1,129 @@
+from btwa_api.app.detection import get_score, detect
+from datetime import datetime, timedelta
+
+
+def test_get_score():
+    # if signal <= 70:
+    assert get_score(0) == 35
+    assert get_score(50) == 35
+    assert get_score(70) == 35
+
+    # if signal <= 75:
+    assert get_score(74) == 25
+    assert get_score(75) == 25
+
+    # if signal <= 80:
+    assert get_score(76) == 17
+    assert get_score(80) == 17
+
+    # if signal <= 85:
+    assert get_score(84) == 10
+    assert get_score(85) == 10
+
+    # if signal <= 90:
+    assert get_score(88) == 5
+    assert get_score(90) == 5
+
+    # else
+    assert get_score(91) == 0
+
+    assert get_score(1022261615165151151) == 0
+    assert get_score(----1022261615165151151) == 0
+    assert get_score(1022261615165151151556565656565665565656) == 0
+
+
+def test_detect():
+    NO_OF_DAYS_TO_COUNT = 14
+
+    ONE_DAY = 1
+    TWO_DAY = 2
+    THREE_DAY = 3
+    FOR_DAY = 4
+    THIRTEEN_DAY = 13
+    FIFTEEN_DAY = 15
+
+    date_n_days_ago = datetime.now() - timedelta(days=ONE_DAY)
```

Comment:
`datetime.now` is repeated a lot. I'd store it into a variable and use this one everywhere `datetime.now()` is called now

Reply:
thanks